documentation/projectVision.md

```diff
@@ -0,0 +1,20 @@
+# VISION
+ Help development teams save their projects from unexpected assembly breakdowns and security issues due to vulnerabilities in third party libraries. 
+# TARGET GROUP
+- Software developers, project managers;
+- Security/code auditors.
+# OBJECTIVES AND GOALS
+- Fast and easily find vulnerabilities in - user’s projects;
+- Have an ability to see statistical data for found vulnerabilities.
+# FEATURES
+- As a user I want to have an opportunity to register and authorization on website;
+- As a user I want to be able to create projects on my account;
```

Comment:
We need to have CRUD on projects, not just 'create' action. I think, it's better to change user story to "...I want to have an ability to create and manage several projects..."